config/field.field.node.plan.field_agency.yml

```diff
@@ -17,13 +16,10 @@ translatable: false
 default_value: {  }
 default_value_callback: ''
 settings:
-  handler: 'default:node'
+  handler: views
   handler_settings:
-    target_bundles:
-      agency: agency
-    sort:
-      field: title
-      direction: ASC
-    auto_create: false
-    auto_create_bundle: ''
+    view:
+      view_name: agencies
+      display_name: entity_reference_1
```

Comment:
I usually try to rename these so they're more portable and easy to work with. No need to hold up this PR though.

Reply:
I can work on that today, I always forget about all the naming fields on Views